docs/bundles/SyliusTaxationBundle/custom_calculators.rst

```diff
@@ -23,3 +23,28 @@ All calculators implement the **TaxCalculatorInterface**. First, you need to cre
             return $amount * ($rate->getAmount() + 0,15 * 0,30);
         }
     }
+
+
+.. note::
+
+    Now we need to register calculator as service.
+
+Remember we set calculator for tax rate as ``app.server_tax``.
+
+.. code-block:: php
+
+    $cloudServerTax = $repository
+            ->createNew()
+            ->setName('Cloud Server Tax')
+            ->setCalculator('app.server_tax')
+            ->setAmount(0,15);
+
+Service should be tagged with key value pair ``name: sylius.tax_calculator`` and ``calculator: app.server_tax``.
+
+.. code-block:: yaml
+
+    #
+    app.tax_calculator.server_tax_calculator:
+        class: AppBundle\Taxes\Calculator\TouristTaxCalculator
```

Comment:
Can we make this example more consistent? A few lines above we have created a `FeeCalculator` and now we are registering a `FeeCalculator` under `app.server_tax` tag.